doc/contact/index.rst

```diff
@@ -0,0 +1,8 @@
+Contact
```

Comment:
I'm not sure this page is needed. I don't really like how this works out:
https://pylint--6613.org.readthedocs.build/en/6613/contact/index.html

Would be better if clicking on `Contact` in the sidebar brings you to the page immediately.